packages/common/src/dto/personal-information.dto.ts

```diff
@@ -1,15 +1,22 @@
-import { IsString, Length, Matches } from 'class-validator';
+import { IsNotEmpty, IsString, Length, Matches } from 'class-validator';
 
 export class PersonalInformationDTO {
   @IsString()
-  @Length(1, 255)
+  @Length(1, 255, { message: 'First Name must be between 1 and 255 characters' })
+  @IsNotEmpty({ message: 'First Name is Required' })
+  @Matches(/[^ ]+/, { message: 'First Name is Required' })
   firstName!: string;
 
   @IsString()
-  @Length(1, 255)
+  @Length(1, 255, { message: 'Last Name must be between 1 and 255 characters' })
+  @IsNotEmpty({ message: 'Last Name is Required' })
+  @Matches(/[^ ]+/, { message: 'Last Name is Required' })
   lastName!: string;
 
   @IsString()
-  @Matches(/^[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][\s-]?\d[ABCEGHJ-NPRSTV-Z]\d$/i)
+  @Matches(/^[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][\s-]?\d[ABCEGHJ-NPRSTV-Z]\d$/i, {
```

Comment:
Total nit, but this regex is a bit hard to read. I wonder if it would be worth breaking up/ simplifying. Maybe having a constant for all the valid characters?

Reply:
Would a link to a source defining it be alright?

Reply:
Totally!